pof/src/types.rs

```diff
@@ -272,6 +272,18 @@ impl Vec3d {
             UpAxis::ZUp => Vec3d { x: self.x, y: self.z, z: self.y },
         }
     }
+    pub fn to_rotation(&self) -> nalgebra_glm::Mat4x4 {
```

Comment:
This might be better termed `to_matrix`? The idea here is to make an orientation matrix purely from a single vector, and I think `to_rotation` is a little more ambiguous.